src/main/java/org/spongepowered/api/text/translation/Translation.java

```diff
@@ -48,24 +49,19 @@
     String getId();
 
     /**
-     * Gets the default translation without any parameters. If the translations
-     * contains any parameters they will be filled with empty text.
+     * Gets the default translation without any parameters replaced.
      *
+     * @param locale The language to get the translated format string for
      * @return The default translation without any parameters
      */
-    String get();
+    String get(Locale locale);
```

Comment:
I think we should keep the getters without language to get it in the default language of the server (which is also the same as used in `Texts.toPlain(Text)` without locale argument)